lib/common/utils.ts

```diff
@@ -569,3 +569,15 @@ export function findEventTask(target: any, evtName: string): Task[] {
   }
   return result;
 }
+
+Zone[zoneSymbol('patchEventTargetMethods')] = patchEventTargetMethods;
```

Comment:
I feel very uneasy with exposing these APIs even if they are behind a symbol. What is the reasoning?